ci/htcondor/Dockerfile

```diff
@@ -0,0 +1,9 @@
+# base image
+ARG  HTCONDOR_VERSION
+FROM htcondor/mini:${HTCONDOR_VERSION}${HTCONDOR_VERSION:+-}el7
```

Comment:
needed to have a HTCondor image. Actually for HTCondor there is close to 0 speedup.